models/video-model.js

```diff
@@ -0,0 +1,42 @@
+const mongoose=require("mongoose");
+const {Schema}=mongoose;
+const videoList=require("./video-data");
+
+const videoSchema=new Schema({
+  vid:{
+    type:String,
+    required:"Video Id is a required attribute",
+    unique:true
+  },
+  thumbnailURL:{
+    type:String,
+    required:"Thumbnail URL is a required attribute"
+  },
+  title:{
+   type:String,
+   required:"title is required"
+  },
+  description:String,
+  channelName:String,
+  published_date:String
+},
+{
+  timestamps:true,
+}
+)
+
+const Video=mongoose.model("Video",videoSchema);
+
+const fillVideosCollection= async ()=> {
+  try {
+    videoList.forEach(async (video) => {
+      const newVideo = new Video(video);
+      const savedVideo = await newVideo.save();
+      console.log(savedVideo);
+    });
+  } catch (e) {
+    console.log(e);
+  }
```

Comment:
You can take this to videos.controller

Reply:
This was to run only once. So I thought to place it in model only